faker/providers/date_time/__init__.py

```diff
@@ -1426,16 +1426,16 @@ def _rand_seconds(self, start_datetime: int, end_datetime: int) -> float:
         ),
         Country(
             timezones=[
-                "Europe/Kiev",
-                "Europe/Uzhgorod",
-                "Europe/Zaporozhye",
+                "Europe/Kyiv",
+                "Europe/Uzhhorod",
```

Comment:
According to the [tz database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), this should still be `"Europe/Uzhgorod"`

Reply:
Could you change this back to `"Europe/Uzhgorod"`?